src/components/va-alert/va-alert.tsx

```diff
@@ -65,6 +66,19 @@ export class VaAlert {
   }
 
   private handleAlertBodyClick(e: MouseEvent): void {
+    let headlineText = null;
+
+    // This is the happy path, meaning the user isn't using IE11
+    try {
+      const children = this.el.shadowRoot.querySelector('slot').assignedNodes();
+      const headlineNode = children[0];
+      headlineText = headlineNode.textContent;
```

Comment:
What if there is no `<h2 slot="headline">Foo</h2>`? Will this throw an exception? (And thus get caught below?)